frontend/ui/widget/magnifier.lua

```diff
@@ -0,0 +1,156 @@
+--[[--
+Widget that displays zoomed pic.
+
+It vanishes on key press or after a given timeout.
+
+Example:
+    local UIManager = require("ui/uimanager")
+    local _ = require("gettext")
+    local Screen = require("device").screen
+    local magnifier
+    magnifier = Magnifier:new{
+        image = Image,
+        height = Screen:scaleBySize(400),
+        width = Screen:scaleBySize(400),
+        timeout = 5,  -- This widget will vanish in 5 seconds.
+    }
+    UIManager:show(magnifier)
+    magnifier:onShowKeyboard()
+]]
+
+local Blitbuffer = require("ffi/blitbuffer")
+local BottomContainer = require("ui/widget/container/bottomcontainer")
+local Topcontainer = require("ui/widget/container/topcontainer")
+local Device = require("device")
+local FrameContainer = require("ui/widget/container/framecontainer")
```

Comment:
Re-order them Alphabetically, following the exceptions you might see in other modules.

Reply:
Done